aten/src/ATen/Declarations.cwrap

```diff
@@ -1928,6 +1928,7 @@
     - Double
   backends:
     - CPU
+    - CUDA
```

Comment:
This is defining a CUDA implementation of _th_histc in our TH backend. We don't want this; the bincount kernel is in our aten backend, and so will the CUDA implementation of histc. Instead, you should add an entry to native_functions.yaml: https://github.com/pytorch/pytorch/blob/68251fb93196a4c28a0c9f9ce37896b21c6c04e0/aten/src/ATen/native/native_functions.yaml#L2903-L2904. It should look something like https://github.com/pytorch/pytorch/blob/68251fb93196a4c28a0c9f9ce37896b21c6c04e0/aten/src/ATen/native/native_functions.yaml#L256-L260 , for example